integration-tests/manifests/cmd/validate_instrumentation_vars.go

```diff
@@ -89,6 +99,15 @@ func verifyInstrumentationEnvVariables(clientset *kubernetes.Clientset, namespac
 	}
 	fmt.Println("JSON data:", jsonData)
 
+	if appSignals == "no_app_signals" {
```

Comment:
Does your test ever go within this if statement?

Reply:
I can add a log for that.